runtime/autoload/man.vim

```diff
@@ -135,7 +135,15 @@ function! s:get_page(path) abort
   let manwidth = empty($MANWIDTH) ? winwidth(0) : $MANWIDTH
   " Force MANPAGER=cat to ensure Vim is not recursively invoked (by man-db).
   " http://comments.gmane.org/gmane.editors.vim.devel/29085
-  return s:system(['env', 'MANPAGER=cat', 'MANWIDTH='.manwidth, 'man', a:path])
+  try
+    if !has('win32') && system('uname -s') =~ 'Darwin'
```

Comment:
There is no need for the win32 check or the try statement, you can remove them. Also, please change the `=~` to `#=~`.